client/verta/verta/_internal_utils/_utils.py

```diff
@@ -260,15 +260,25 @@ def email(self):
         else:
             return None
 
-    def _get_visible_orgs(self):
+    def _get_visible_workspaces(self):
         response = self.make_proto_request(
             "GET", "/api/v1/uac-proxy/workspace/getVisibleWorkspaces"
         )
         response = self.must_proto_response(response, Workspace_pb2.Workspaces)
 
-        org_names = map(lambda workspace: workspace.org_name, response.workspace)
-        org_names = filter(None, org_names)
-        return list(org_names)
+        workspace_names = map(lambda workspace: "{}:{}".format(workspace.org_id, workspace.org_name), response.workspace)
+        workspace_names = filter(None, workspace_names)
+        return list(workspace_names)
+
+    def _get_organization_id(self):
+        response = self.make_proto_request(
+            "GET", "/api/v1/uac-proxy/workspace/getVisibleWorkspaces"
+        )
+        response = self.must_proto_response(response, Workspace_pb2.Workspaces)
+
+        workspace_names = map(lambda workspace: workspace.org_id, response.workspace)
+        workspace_names = filter(None, workspace_names)
+        return list(workspace_names)[0]
```

Comment:
Is it possible for `getVisibileWorkspaces` to return workspaces in other organizations? `list(workspace_names)[0]` seems like it will basically return the `org_id` of a random workspace, so I'm worried that if we have a multi-organization setup, we might get the wrong ID 🤔 but I'm also not sure how permissions v2 works 😅